component/src/main/java/org/wso2/extension/siddhi/io/ibmmq/source/IBMMQSource.java

```diff
@@ -81,6 +77,11 @@
                         type = DataType.STRING,
                         optional = true,
                         defaultValue = "null"),
+                @Parameter(name = IBMMQConstants.WORKER_COUNT,
+                        description = "Number of worker threads listening on the given queue.",
```

Comment:
Say when multiple workers are enabled even ordering is not preserved

Reply:
Fixes added with: 468d4c0a159cab795a5a33234b5aaedacd452618